sdk/include/ndk/halfuncs.h

```diff
@@ -32,25 +32,120 @@ Header Name:
+#define HalRequestInterrupt                                 HALPRIVATEDISPATCH->HalRequestInterrupt
+#define HalEnumerateUnmaskedInterrupts                      HALPRIVATEDISPATCH->HalEnumerateUnmaskedInterrupts
+#define HalFlushAndInvalidatePageExternalCache              HALPRIVATEDISPATCH->HalFlushAndInvalidatePageExternalCache
+#define KdEnumerateDebuggingDevices                         HALPRIVATEDISPATCH->KdEnumerateDebuggingDevices
+#define HalFlushIoRectangleExternalCache                    HALPRIVATEDISPATCH->HalFlushIoRectangleExternalCache
+#define HalPowerEarlyRestore                                HALPRIVATEDISPATCH->HalPowerEarlyRestore
+#define HalQueryCapsuleCapabilities                         HALPRIVATEDISPATCH->HalQueryCapsuleCapabilities
+#define HalUpdateCapsule                                    HALPRIVATEDISPATCH->HalUpdateCapsule
+#define HalPciMultiStageResumeCapable                       HALPRIVATEDISPATCH->HalPciMultiStageResumeCapable
+#define HalDmaFreeCrashDumpRegisters                        HALPRIVATEDISPATCH->HalDmaFreeCrashDumpRegisters
+#define HalAcpiAoacCapable                                  HALPRIVATEDISPATCH->HalAcpiAoacCapable
+#define HalInterruptSetDestination                          HALPRIVATEDISPATCH->HalInterruptSetDestination
+#define HalGetClockConfiguration                            HALPRIVATEDISPATCH->HalGetClockConfiguration
+#define HalClockTimerActivate                               HALPRIVATEDISPATCH->HalClockTimerActivate
+#define HalClockTimerInitialize                             HALPRIVATEDISPATCH->HalClockTimerInitialize
+#define HalClockTimerStop                                   HALPRIVATEDISPATCH->HalClockTimerStop
+#define HalClockTimerArm                                    HALPRIVATEDISPATCH->HalClockTimerArm
+#define HalTimerOnlyClockInterruptPending                   HALPRIVATEDISPATCH->HalTimerOnlyClockInterruptPending
+#define HalAcpiGetMultiNode                                 HALPRIVATEDISPATCH->HalAcpiGetMultiNode
+#define HalPowerSetRebootHandler                            HALPRIVATEDISPATCH->HalPowerSetRebootHandler
+#define HalIommuRegisterDispatchTable                       HALPRIVATEDISPATCH->HalIommuRegisterDispatchTable
+#define HalTimerWatchdogStart                               HALPRIVATEDISPATCH->HalTimerWatchdogStart
+#define HalTimerWatchdogResetCountdown                      HALPRIVATEDISPATCH->HalTimerWatchdogResetCountdown
+#define HalTimerWatchdogStop                                HALPRIVATEDISPATCH->HalTimerWatchdogStop
+#define HalTimerWatchdogGeneratedLastReset                  HALPRIVATEDISPATCH->HalTimerWatchdogGeneratedLastReset
+#define HalTimerWatchdogTriggerSystemReset                  HALPRIVATEDISPATCH->HalTimerWatchdogTriggerSystemReset
+#define HalInterruptVectorDataToGsiv                        HALPRIVATEDISPATCH->HalInterruptVectorDataToGsiv
```

Comment:
```suggestion
#endif
#if (NTDDI_VERSION >= NTDDI_WINTHRESHOLD)
#define HalInterruptVectorDataToGsiv                        HALPRIVATEDISPATCH->HalInterruptVectorDataToGsiv
```
or using `NTDDI_WIN10` as I suggested elsewhere.